### PR TITLE
A script to reset the basic auth secret.

### PR DESCRIPTION
**Pull request by jlewi**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/2662)

### CODE REVIEW

File: scripts/create_password_secret.sh [hunk 1]
```diff
@@ -8,7 +8,15 @@ user=$1
...
+#PASSWORDB64="${password}"
+
+set +e
+kubectl -n kubeflow delete secret kubeflow-login
+set -e
+
+kubectl -n kubeflow create secret generic kubeflow-login --from-literal=username=${USER} --from-literal=passwordhash="${PASSWORDB64}"
 
 # Delete the pod to load the new secret
 kubectl -n kubeflow delete pods -l app=basic-auth-login
```
Comment (kunmingg) [inline scripts/create_password_secret.sh]: We don't need to change the pod serving our login UI.

**Comment by jlewi** [comment]
Comments addressed; @kunmingg PTAL

**Comment by jlewi** [comment]
@kunmingg Could you take another look please?

**Pull request merged**